diff -urN goobox-0.7.0.orig/src/actions.c goobox-0.7.0/src/actions.c
--- goobox-0.7.0.orig/src/actions.c	2005-01-08 22:43:23.581042320 +0100
+++ goobox-0.7.0/src/actions.c	2005-01-08 22:45:34.928074536 +0100
@@ -134,10 +134,10 @@
 		"Paolo Bacchilega <paolo.bacchilega@libero.it>", 
 		NULL
 	};
-	/*
-	  const char       *documenters [] = {
-	  NULL
-	  };*/
+	
+	const char       *documenters [] = {
+		NULL
+	};
 	const char       *translator_credits = _("translator_credits");
 
 	if (about != NULL) {
@@ -154,7 +154,7 @@
 		      "copyright", "Copyright \xc2\xa9 2004 Free Software Foundation, Inc.",
 		      "comments", _("CD player and ripper"),
 		      "authors", authors,
-		      /*"documenters", documenters,*/
+		      "documenters", documenters,
 		      "translator_credits", strcmp (translator_credits, "translator_credits") != 0 ? translator_credits : NULL,
 		      "website", NULL,
 		      "website_label", NULL,
